components/support/migration/src/main/java/mozilla/components/support/migration/FennecMigrator.kt

```diff
@@ -334,6 +371,9 @@ class FennecMigrator private constructor(
             logger.debug("Migrated bookmarks.")
             Result.Success(Unit)
         } catch (e: Exception) {
+            crashReporter.submitCaughtException(
+                FennecMigratorException.MigrateBookmarksException(e::class.qualifiedName)
```

Comment:
Instead of the `name` (String) parameter... doesn't it make more sense to set the `cause` (Exception) parameter so that we get a nice stacktrace including the stacktrace of the exception that happened here? Much more helpful and doesn't require reflection? :)

Reply:
Done; i've decided to not block on the PII thing right now, and deal with it more generally in a follow-up: https://github.com/mozilla-mobile/android-components/issues/5095